src/plugins/event-plugins/JsErrorPlugin.ts

```diff
@@ -6,26 +6,31 @@ export const JS_ERROR_EVENT_PLUGIN_ID = 'com.amazonaws.rum.js-error';
 
 export type PartialJsErrorPluginConfig = {
     stackTraceLength?: number;
+    errorsToExclude?: RegExp[];
 };
 
 export type JsErrorPluginConfig = {
     stackTraceLength: number;
+    errorsToExclude?: RegExp[];
 };
 
 const defaultConfig: JsErrorPluginConfig = {
-    stackTraceLength: 200
+    stackTraceLength: 200,
+    errorsToExclude: []
 };
 
 export class JsErrorPlugin implements Plugin {
     private pluginId: string;
     private enabled: boolean;
     private config: JsErrorPluginConfig;
+    private ignoredErrors: Set<string>;
```

Comment:
If the customer has many regex inputs in `errorsToExclude`, I believed we could cache the errors that were ignored already and simply speed up the checking without having to go through the entire list at the expense of using space.

After doing some research, it seems that iterating through hundreds of regex patterns seems to be still very fast, so this may not be necessary at all. What do you think?